Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -57,11 +57,11 @@ define([
          * The following properties are part of the {@link Cesium3DTileContent} interface.
          */
         this.state = Cesium3DTileContentState.UNLOADED;
-        this.contentReadyToProcessPromise = when.defer();
-        this.readyPromise = when.defer();
         this.batchTableResources = undefined;
         this.featurePropertiesDirty = false;
 
+        this._contentReadyToProcessPromise = when.defer();
+        this._readyPromise = when.defer();
```

Comment:
OK, we can always revisit it in a separate PR; probably when we do the big `when` -> `bluebird` switchover.